.circleci/config.yml

```diff
@@ -367,18 +318,18 @@ workflows:
             - build-align
       - build-make-docker:
           requires:
-            - start
-      - build-integration1:
-          requires:
-            - start
-      - build-integration2:
-          requires:
-            - start
-      - finish:
+            - build-align
+      - test-integration:
+          name: "test-integration-<<matrix.pdk>>(<<matrix.design>>)"
           requires:
             - build-align
-            - build-cktgen
-            - build-PlaceRouteHierFlow
-            - build-integration1
-            - build-integration2
+          matrix:
+            parameters:
+              pdk:
+                - "Finfet"
+              design:
+                - "BUFFER or ota"
+                - "San or BTSW"
+                - "not SAR and not San and not BUFFER and not DLL and not ota and not single_to_differential_converter and not COMPARATOR_2LEVEL_BIDIRECTIONAL_MAC_SKEW and not CTD"
+                # - "ota and not OTA_FF_2s_v3e"
```

Comment:
Can you review this list to ensure the commented lines can be completely removed?

These are the testcases being run for every commit.